src/hotspot/share/runtime/mutexLocker.hpp

```diff
@@ -70,14 +70,14 @@ extern Monitor* Threads_lock;                    // a lock on the Threads table
                                                  // (also used by Safepoints too to block threads creation/destruction)
 extern Mutex*   NonJavaThreadsList_lock;         // a lock on the NonJavaThreads list
 extern Mutex*   NonJavaThreadsListSync_lock;     // a lock for NonJavaThreads list synchronization
-extern Monitor* CGC_lock;                        // used for coordination between
+extern Monitor* G1CGC_lock;                        // used for coordination between
```

Comment:
In all hunks in this file: comment indenting. May even add "G1" to comments, where appropriate.

Reply:
I remember fixing the indentation, maybe it got lost somewhere. Thanks for pointing this out.